frontend/src/components/error/snackBar.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
```

Comment:
I think it would be better to make the component folder "snackbar" and not "error". Then, in src/components/snackbar, have a base snackbar (BaseSnackbar), then you can have things like ErrorSnackbar, InfoSnackbar, etc. that derive from it. You can use dropdowns and modals as examples for this (I recommend dropdowns, the PR to refactor modals is still stuck in hkn-member-portal so it's a bit hard to read through modals here in hkn-server).

If we do it this way, devs can just use ErrorSnackbar to display errors, InfoSnackbar to display info and not have to worry about customizing their snackbar. The point is that with these abstractions, we want to limit customization where we can, in order to have some sort of standardization in terms of these components (i.e. we don't want different ErrorSnackbar to look and be displayed differently from each other)

Let me know what you think about this.

Reply:
Yah that sounds reasonable I'll rename them to something more appropriate